ads/google/imaVideo.js

```diff
@@ -519,6 +519,7 @@ export function playAds(global) {
 export function onContentEnded() {
   contentComplete = true;
   adsLoader.contentComplete();
+  window.parent./*OK*/postMessage({event: VideoEvents.PAUSE}, '*');
```

Comment:
didn't notice that we're sending raw postmessages. do we plan eventually migrate to the unified messaging format that IframeMessagingClient is enforcing?